script_404.js

```diff
@@ -11,6 +11,9 @@ const routes = {
 function loadContent() {
   const path = window.location.hash.substring(1);
   const page = routes[path];
+  
+  console.log("Current path:", path);
+  console.log("Resolved page:", page);
```

Comment:
Remove console.log or change it with console.info

```suggestion
  console.info("Current path:", path);
  console.info("Resolved page:", page);
```

Reply:
I've changed to console.info